learning/step_by_step/your_first_game.rst

```diff
@@ -79,7 +79,7 @@ are not selectable."
 
 .. image:: img/lock_children.png
 
-Save the scene (click Scene -> Save, or press ``Meta-s``).
+Save the scene (click Scene -> Save, or press ``Control-s`` on Windows/Linux or ``Command-s`` on Mac).
```

Comment:
Good call! The menu used to say `Meta-s`. I'm glad it's changed.

Reply:
Another small thing I noticed: the menus use capitals and `+` signs now too: ``Command+S``

Reply:
I'll change these to use the `+` sign.